apps/frontend/src/index.tsx

```diff
@@ -31,6 +32,10 @@ createRoot(document.querySelector("#root") as HTMLElement).render(
 						element: <App />,
 						path: AppRoute.ROOT,
 					},
+					{
+						element: <PrivateRoute />,
+						path: AppRoute.APP,
+					},
```

Comment:
it should look like this
```suggestion
element: <App />,
path: AppRoute.APP,
},
					{
						element: <Landing />,
						path: AppRoute.ROOT,
					},
```